project/AppDependencies.scala

```diff
@@ -2,28 +2,31 @@ import sbt._
 
 object AppDependencies {
 
-  private val bootstrapVersion = "7.19.0"
+  private val bootstrapVersion = "8.4.0"
   val dispatchVersion        = "1.2.0"
-  lazy val mongoVersion = "1.3.0"
+  lazy val mongoVersion = "1.7.0"
 
 
-  val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % bootstrapVersion,
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-work-item-repo-play-28" % mongoVersion,
+  val compile: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-30"  % bootstrapVersion,
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-work-item-repo-play-30" % mongoVersion,
     // required for scalaxb
-    "org.scala-lang.modules"  %% "scala-xml"                  % "1.3.0",
     "org.scala-lang.modules"  %% "scala-parser-combinators"   % "1.1.2",
     "javax.xml.bind"          % "jaxb-api"                    % "2.3.1",
     "org.dispatchhttp"        %% "dispatch-core"              % dispatchVersion,
-    "com.beachape"            %% "enumeratum-play-json"       % "1.6.0",
+    "com.beachape"            %% "enumeratum-play-json"       % "1.8.0",
     "org.typelevel"           %% "cats-core"                  % "2.10.0"
 
   )
 
-  val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"     % bootstrapVersion,
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"    % mongoVersion,
+  val test: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"             %% "bootstrap-test-play-30"     % bootstrapVersion,
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-30"    % mongoVersion,
     "org.scalatestplus"       %% "scalatestplus-mockito"      % "1.0.0-M2",
     "org.mockito"             %% "mockito-scala"              % "1.17.12"
-  ).map(_ % "test, it")
+  ).map(_ % "test")
+
+  val overrides: Seq[ModuleID] = Seq(
```

Comment:
What's this for, out of interest

Reply:
I think the new play-30 depends from scala.2.13 V2.2.0, but play bring in a lower version of scala-xml, so we need to override this to get the right version